### ISSUE TITLE
[BUG] alt text tag in jumbotrons does not work

### ISSUE THREAD

**Issue by jdhalbert** [uweb/uw_wp_theme#94]
## Describe the bug

Jumbotron shortcodes do not respect the "alt" tag for images within them.  

## Describe your environment

Client: Brave 1.79.126 on Windows 11
Server: Wordpress 6.8.1 with this theme running on a UW Standard Managed Server

## Steps to reproduce

1. Create a Wordpress page with a jumbotron shortcode containing an `alt` tag
2. Publish the page
3. Inspect the page to see no corresponding `alt` tag in the page source
4. Alternatively, see and inspect the several examples of this issue apparent on the UW [shortcode cookbook](https://www.washington.edu/docs/shortcode-cookbook/jumbotrons/) page. The examples all contain an `alt` tag in the example shortcode text but their images are missing `alt` tags in the page source.

## Expected behavior

The `alt` tag in the jumbotron shortcode could result in an alt text tag on the jumbotron's image. Also, jumbotron images should maybe default to being labeled as decorative if explicit alt text is not provided (`alt=""`)

## Current Behavior

The `alt` tag in jumbotron shortcode does not seem to have any effect.

## Possible Solution

The `alt` attribute seems to be missing from the [jumbotron shortcode implementation](https://github.com/uweb/uw_wp_theme/blob/58837617a31be2fcb0f7bcc6cf579123f7ee39ca/inc/shortcodes/class.jumbotron-shortcode.php#L17).

## Screenshots / Video

N/A (see linked examples)

## Related Issues and/or PRs

Searched for related issues but could not find any.

**Comment by tjsuw** [comment]
This has been resolved as of version [3.8](https://github.com/uweb/uw_wp_theme/releases/tag/3.8.0).
